### PR TITLE
Lot's of stuff :(

### PR DESCRIPTION
**Pull request by dodalovicgran**

### CODE REVIEW

**Comment by dodalovicgran** [comment]
@gglazewskigran Sorry for this big PR

File: src/database.rs [hunk 1]
```diff
@@ -8,19 +8,32 @@ pub struct Db {
 
 impl Db {
     pub fn new() -> Self {
+        let clients = ["Sony", "Google", "Amazon"];
```
Comment (gglazewskigran) [inline src/database.rs]: Definition of a client:
A person or organization using the services of a lawyer or other professional person or company.

I wouldn't use the word "client" as those entities are not clients in this context. Maybe an employer, a company, a recruiter?

File: src/jobs/handlers.rs [hunk 2]
```diff
@@ -41,17 +43,33 @@ async fn new_job_view() -> impl Responder {
...
     new_job: web::Form<NewJob>,
     db_mutex: web::Data<Mutex<Db>>,
 ) -> impl Responder {
-    db_mutex
-        .lock()
-        .unwrap()
-        .add(Job::new(new_job.into_inner().title));
+    let new_job = new_job.into_inner();
+    db_mutex.lock().unwrap().add(new_job.into());
+    //db_mutex.lock().unwrap().add(Job::from(new_job));
```
Comment (gglazewskigran) [inline src/jobs/handlers.rs]: please remove the comment

File: static/root/css/style.css [hunk 3]
```diff
@@ -1 +1,60 @@
...
+nav ul {
+    display: flex;
+    /* 5 */
+    list-style: none;
+    /* 6 */
+}
+
+nav li {
+    padding-left: 1rem;
+    /* 7! */
```
Comment (gglazewskigran) [inline static/root/css/style.css]: Please remove the comments

File: src/jobs/handlers.rs [hunk 4]
```diff
@@ -41,17 +43,33 @@ async fn new_job_view() -> impl Responder {
...
     Redirect::to("/").using_status_code(StatusCode::SEE_OTHER)
```
Comment (gglazewskigran) [inline src/jobs/handlers.rs]: You can use as well:
`Redirect::to("/").see_other()`

File: templates/job.stpl [hunk 5]
```diff
@@ -1,8 +1,27 @@
...
+          <input type="text" name="title" disabled value="<%= job.title %>"><br>
+          <label for="title">Type:</label>
+          <input type="text" name="type" disabled value="<%= job.job_type | dbg %>"><br>
+          <label for="start">Start:</label>
+          <input type="text" name="start" disabled value="<%= job.start %>"><br>
+          <label for="location">Location:</label>
+          <input type="text" name="location" disabled value="<%= job.location | dbg %>"><br>
+          <label for="client">Client:</label>
+          <input type="text" name="client" disabled value="<%= job.client %>"><br>
+        </fieldset>
```
Comment (gglazewskigran) [inline templates/job.stpl]: This shouldn't be a form. Maybe a table or just some divs, but not a form.

**Review by gglazewskigran — CHANGES_REQUESTED** [review]
A great PR! We are moving forward. I wasn't expecting you to know advanced CSS!

I'd ask for little changes:
1. Naming - "client" is confusing.
2. JobDetails shouldn't use form fields.

**Review by gglazewskigran — APPROVED** [review]
:rocket:

**Pull request merged**